etc/taskcluster/decision_task.py

```diff
@@ -868,8 +868,8 @@ def magicleap_build_task(name, build_type):
             path="magicleap"
         )
         .with_directory_mount(
-            "https://servo-deps.s3.amazonaws.com/magicleap/TempSharedCert.zip",
-            sha256="cdc2d26bc87ecf1cd8133df4e72c4eca5df7ddd815d0adf3045460253c1fe123",
+            "https://servo-deps.s3.amazonaws.com/magicleap/ServoCICert-expires-2020-08-21.zip",
```

Comment:
Clever!

Reply:
Hopefully this will make it obvious to the next poor soul what the error is. That poor soul is probably future!me.